fe/fe-core/src/main/java/com/starrocks/server/Session.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.server;
+
+import com.google.api.client.util.Lists;
+import com.google.common.collect.Maps;
+import org.apache.arrow.util.Preconditions;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+public class Session {
```

Comment:
Which a session means? A connect context life cycle or a query process life cycle?

Reply:
> Which a session means? A connect context life cycle or a query process life cycle?

this is a temporary pr, I will split to multi prs, plz ignore it.